spec/spec_helper.rb

```diff
@@ -1,19 +1,23 @@
-require "codeclimate-test-reporter"
-CodeClimate::TestReporter.start
-
+require 'dotenv'
 require 'rspec/given'
 require 'webmock/rspec'
 
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 require 'sherlock_homes'
 
+Dotenv.load('.env.test')
```

Comment:
I'll merge this anyway, but it seems like we may want Dotenv.load to happen before we require the rest of the gem.  Perhaps not if we're autoload-ing the config.  Just something to keep in mind, if we find that the tests aren't pulling the keys.